src/shared/constants/apiEndpoints.ts

```diff
@@ -1,2 +1,2 @@
 // This folder will contain or hold constants. For example in this particular file we can store the api endpoints as constants
-export const CLI_COMMAND_URL="http://localhost:8080/cli"
\ No newline at end of file
+export const ROOT_URL = "http://localhost:8080/";
```

Comment:
@tarun-29 can you please make changes to handle these via env or configs?

Reply:
this is resolved

Reply:
Is this used anywhere now?
If not please remove this file and have a .env file that can take playground mono URL which points to API URL.
Let's name the var as `PLAYGROUND_MONO_URL`.

Reply:
@lucifercr07 This is resolved